src/routes/data/get.rs

```diff
@@ -71,13 +75,27 @@ pub fn without_token<S: SessionStore, R: Renderer, T: TokenGenerator>(
                   session_with_store: SessionWithStore<S>,
                   token: String,
                   render_engine: R| async move {
+
+                let mut sanitized_message = None;
```

Comment:
Instead of declaring sanitized_message here and making it mut, there's a more functional way of doing this using and_then:
```
let sanitized_message = query_params.js_message.and_then(|m| {
    let decoded_param = ...
});
```

Reply:
done